_includes/pbc/all/install-features/202410.0/install-the-amazon-quicksight-module.md

```diff
@@ -0,0 +1,596 @@
+| QuicksightUserSyncCreateConsole | Persists in `spy_quicksight_user` DB table users registered in quicksight by persisted user emails.      |               | SprykerEco\Zed\AmazonQuicksight\Communication\Console |
+
+**src/Pyz/Zed/Console/ConsoleDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Console;
+
+use Spryker\Zed\Console\ConsoleDependencyProvider as SprykerConsoleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use SprykerEco\Zed\AmazonQuicksight\Communication\Console\QuicksightUserSyncCreateConsole;
+
+class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return list<\Symfony\Component\Console\Command\Command>
+     */
+    protected function getConsoleCommands(Container $container): array
+    {
+        return [
+            new QuicksightUserSyncCreateConsole(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+To check the `quicksight-user:sync:create` console command:
+
+1. Create new Quicksight user in Quicksight.
+2. Run the following command:
+```bash
+console quicksight-user:sync:create
+```
+3. In the `spy_quicksight_user` table check that the corresponding Quicksight user has been added.
+
+{% endinfo_block %}
+
+{% info_block infoBox "Deployment" %}
+
+In order users to be synchronized automatically during deployment, you can configure the installation stage of the pipeline, for example the `destructive` pipeline. Add the following command to the end of the `demodata` section:
+
+**config/install/destructive.yml**
+```json
+sections:
+    demodata:
+#       ...other commands
+        create-quicksight-users:
+            command: 'vendor/bin/console quicksight-user:sync:create'
+```
+
+{% endinfo_block %}
+
+## Install feature frontend
+
+Follow the steps below to install the Amazon Quicksight module frontend.
+
+### Prerequisites
+
+Install the required features:
+
+| NAME         | VERSION          | INSTALLATION GUIDE                                                                                                                                          |
+|--------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html) |
+
+### 1) Enable Javascript and CSS changes:
+
+1. Update `package.json`:
+
+**package.json**
+```json
+{
+    "workspaces": [
+        "vendor/spryker-eco/*/assets/Zed"
+    ],
+}
+```
+
+2. Run the following command to enable Javascript and CSS changes: 
+
+```bash
+npm install
+console frontend:project:install-dependencies
+console frontend:zed:build
+```
```

Comment:
missing verification section. How to ensure that assets from eco enabled? Maybe we need to open some page to check that build was successful?